tests/parser/FieldPropsTests.cpp

```diff
@@ -2341,3 +2341,158 @@ COPYREG
         BOOST_CHECK_EQUAL(permx[g], permy[g]);
     }
 }
+
+BOOST_AUTO_TEST_CASE(OPERATE_RADIAL_PERM) {
+    std::string deck_string = R"(
+GRID
+
+PORO
+   6*1.0 /
+
+OPERATE
+    PORO   1  3   1  1   1   1  'MAXLIM'   PORO 0.50 /
+    PORO   1  3   2  2   1   1  'MAXLIM'   PORO 0.25 /
+/
+
+
+PERMR
+   6*1/
+
+PERMTHT
+   6*1000/
+
+OPERATE
+    PERMR   1  3   1  1   1   1  'MINLIM'   PERMR 2 /
+    PERMR   1  3   2  2   1   1  'MINLIM'   PERMR 4 /
+    PERMTHT   1  3   1  1   1   1  'MAXLIM'   PERMTHT 100 /
+    PERMTHT   1  3   2  2   1   1  'MAXLIM'   PERMTHT 200 /
+    PERMZ   1  3   1  1   1   1  'MULTA'    PERMTHT 2 1000 /
+    PERMZ   1  3   2  2   1   1  'MULTA'    PERMR 3  300 /
+/
+
+
+
+
+)";
+
+    UnitSystem unit_system(UnitSystem::UnitType::UNIT_TYPE_METRIC);
+    auto to_si = [&unit_system](double raw_value) { return unit_system.to_si(UnitSystem::measure::permeability, raw_value); };
+    EclipseGrid grid(3,2,1);
+    Deck deck = Parser{}.parseString(deck_string);
+    FieldPropsManager fpm(deck, Phases{true, true, true}, grid, TableManager());
+    const auto& poro = fpm.get_double("PORO");
+    BOOST_CHECK_EQUAL(poro[0], 0.50);
+    BOOST_CHECK_EQUAL(poro[3], 0.25);
+
+    const auto& permx = fpm.get_double("PERMX");
+    BOOST_CHECK_EQUAL(permx[0], to_si(2));
+    BOOST_CHECK_EQUAL(permx[3], to_si(4));
+
+    const auto& permy = fpm.get_double("PERMY");
+    BOOST_CHECK_EQUAL(permy[0], to_si(100));
+    BOOST_CHECK_EQUAL(permy[3], to_si(200));
+
```

Comment:
I would add `BOOST_CHECK( permy == fpm.get_double("PERMTH")` and the same for `PERMX / PERMR`